src/components/20-molecules/footer-small/index.js

```diff
@@ -10,45 +10,40 @@ class AXAFooterSmall extends LitElement {
 
   static get properties() {
     return {
-      languageitems: { type: Array },
-      disclaimeritems: { type: Array },
-      copyrighttext: { type: String },
+      languageItems: { type: Array },
```

Comment:
Having attributes as camel case in not nice. Better in lower case. Why did u change it from `languageitems ` to `languageItems `? can u still add attribute `<axa-footer languageitems="..." ></axa-footer>`? if not, please revert camel case

Reply:
Yes, that's why I did it. Marco gave me this hint. This way we are compliant with the standard from html AND js. :)